dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/storage/GitHubJsonStore.java

```diff
@@ -160,10 +159,13 @@ public Set<Map.Entry<String, JsonElement>> parseStoreToImagesMap(DockerfileGitHu
         }
 
         JsonElement imagesJson = json.getAsJsonObject().get("images");
+        log.info(imagesJson.toString());
+        System.exit(0);
```

Comment:
Is this a remnant of debugging?

Reply:
removed this in the last commit

Reply:
that didn't get pushed because of the squashed commit. just did that